saleor/graphql/schema.graphql

```diff
@@ -3203,19 +3203,21 @@ type VoucherTranslation implements Node {
 }
 
 enum VoucherType {
-  VALUE
```

Comment:
I am not sure that you should remove this field. It should be marked as a deprecated also. As you did with PRODUCT, COLLECTION, CATEGORY

Reply:
If someone has own version of dashboard then it will be incompatible with these changes.